### PR TITLE
Image gallery client side stub

### PR DESCRIPTION
**Pull request by alex-r-bigelow**
Add the basic image gallery (without histograms) from the old site

### CODE REVIEW

**Comment by brianhelba** [comment]
About tests and code style:
* We can `/* eslint-disable */` the entire JS file for the time being. Forget about non-tested style conventions for old code now too.
* I'm willing to let the Stylus style slide a bit for now, but it shouldn't be too hard to convert some of these repeated selectors to use nested Stylus functionality (and the cleaner Stylus syntax) soonish.
* The Jade is simple enough that we can demand it having clean style.

**Comment by brianhelba** [comment]
@alex-r-bigelow What's the status of this? I've tried deploying it live, and the right-side doesn't render at all. Is that intended at this point?

Also, note I merged the latest master (with some important changes) back into this branch.

**Comment by alex-r-bigelow** [comment]
Hmm... there shouldn't *be* a right side...

I thought it was ready to go, but clearly not. I'll see if the new merge with master broke anything.

File: web_external/js/views/body/ImagesView.js [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+          return self.imageCache[d].src;
+        },
+        'preserveAspectRatio': 'xMinYMin',
+        'x': width / 2, // Start new images at the middle of the
+        'y': jQuery('#preview').height() // bottom of the screen
+      })
+      .on('click', function (d) {
+        // TODO: show a lightbox
+        // For now, jump to the item in girder
+        window.open('#item/' + d);
```
Comment (brianhelba) [inline web_external/js/views/body/ImagesView.js]: Let's make this point at `/girder#item/`, now that https://github.com/ImageMarkup/isic-archive/pull/99 is merged.

Comment (brianhelba) [inline web_external/js/views/body/ImagesView.js]: You'll need to merge master yet again, to incorporate this change though.

**Comment by brianhelba** [comment]
@alex-r-bigelow Give me a heads up, and I can temporarily deploy this branch to the live site to see how it works. Also, once you're finished, can you upload a full-viewport screenshot of how it renders on your environment?

**Comment by alex-r-bigelow** [comment]
This is how it looks on isic-archive.upstream



Are you not seeing the same thing?

-Alex

**Comment by alex-r-bigelow** [comment]
It should be ready to go...

**Pull request merged**